iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -0,0 +1,178 @@
+// Copyright 2019 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import interact from 'interactjs'
+import $ from 'jquery'
+import { iXBRLReport } from "./report.js";
+import { Viewer } from "./viewer.js";
+import { Inspector } from "./inspector.js";
+
+export function iXBRLViewer() {
+    this._plugins = [];
+    this.inspector = new Inspector(this);
+    this.viewer = null;
+}
+
+/*
+ * Adds a plugin to the viewer.  The plugin should be an object with one or
+ * more of the methods listed below, which will be called by the viewer.
+ *
+ * preProcessiXBRL(bodyElement, docIndex)
+ *
+ * Called upon viewer intialisation, once for each iXBRL document.  bodyElement
+ * is a DOM object for the body element.  docIndex is the index of the document
+ * within the document set.
+ *
+ * updateViewerStyleElement(styleElts)
+ *
+ * styleElts is a JQuery object consisting of the viewer style elements for
+ * each document in the document set.  Additional CSS can be appended to the
+ * contents, or additional header elements inserted relatie to the provided
```

Comment:
relatie => relative

Reply:
Fixed in 3b5bd130f29c897acd9d3fc26a2eb88016a09653